test/acceptance/acceptance.js

```diff
@@ -55,7 +113,72 @@ describe('Lambda Wrapper', () => {
             await basicTest('sanity', input);
         });
         it('works with epsagon labels', async () => {
-            await basicTest('sanity', input);
+            await basicTest('labels', input);
+        });
+    });
+
+    const syncOptions = ['sync'];
+    const paramOptions = ['no', '1', '2', '3'];
+    const returnOptions = [
+        'simple_return', 'no_return',
+        'succeed', 'fail', 'done',
+        'callback', 'callback_error', 'callback_then_fail'];
+    if (RUNTIME === 'nodejs8.10') {
```

Comment:
Maybe on node6.10 push promise-void or something whereyou return a promise but expect the result to be null

Reply:
then it is like no returning anything which we already check..